app/helpers/review_helper.rb

```diff
@@ -1,8 +1,10 @@
 module ReviewHelper
   def section_begun?(model, section, step_process)
-    fields = retrieve_section_forms(section, step_process).then { retrieve_fields_from_forms(_1) }
-
-    model.slice(fields).values.any? { |value| !value.nil? }
+    retrieve_section_forms(section, step_process)
+      .map { |form_class| form_class.load_form(model) }
+      .map(&:values)
+      .flatten
+      .any? { |value| !value.nil? }
```

Comment:
I know this was pre-existing code, but isn't this just a
```suggestion
      .any?
```

Reply:
Oh yeah, I just noticed that any `false` model value wouldn't pass the `any?` default criteria.